docs/src/environments/_index.md

```diff
@@ -186,3 +186,35 @@ Staging
   Developer2
     Ticket-395
 ```
+
+## Paused environments
+
+Development environments are often used for a limited time and then abandoned.
+To prevent unnecessary consumption of resources,
+Platform.sh automatically pauses development environments where code hasn't been pushed in 15 days.
+
+You can resume a paused environment at any time.
```

Comment:
We could explain that "Pushing new code automatically resume the environment, which can also be resume at any time"

When you are push